news/index.md

```diff
@@ -1,20 +1,43 @@
 ---
 title: News
-layout: default
+layout: news
 permalink: /news/
 ---
+{% raw %}
```

Comment:
Can we move this logic into the  `news.html` template? Or are we using that elsewhere?

Reply:
we can move out this logic into the template

Reply:
done!